ldes-server-domain/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/domain/ldes/retentionpolicy/creation/RetentionPolicyCreatorImpl.java

```diff
@@ -24,13 +26,12 @@ public List<RetentionPolicy> createRetentionPolicyListForView(ViewSpecification
 
 	private RetentionPolicy getRetentionPolicy(RetentionConfig retentionConfig) {
```

Comment:
The method above could be on the ViewSpecification

Reply:
Currently, we treat fragmentation strategies and retention policies in a similar way. That is that an interesting party could develop its own fragmentation strategy or its own retention policy. In that sense, it might be interesting to offload the entire creation of retention policies actually to separate modules which are responsible for creating their retention policy implementation.

This being said the current implementation is a bit 'vlees noch vis', because it doesn't offload the creation nor does it envelop the creation of the retention policy (as I expect is what you mean by your comment). Instead it floats a bit between both.